src/goRunTestCodelens.ts

```diff
@@ -9,29 +9,28 @@ import vscode = require('vscode');
 import path = require('path');
 import { CodeLensProvider, TextDocument, CancellationToken, CodeLens, Command } from 'vscode';
 import { getTestFunctions } from './goTest';
+import { getTestEnvVars } from './util';
 import { GoDocumentSymbolProvider } from './goOutline';
 
 export class GoRunTestCodeLensProvider implements CodeLensProvider {
 	private readonly debugConfig: any = {
 				'name': 'Launch',
 				'type': 'go',
 				'request': 'launch',
-				'mode': 'test',
-				'env': {
-					'GOPATH': process.env['GOPATH']
```

Comment:
I removed because of [this line](https://github.com/goenning/vscode-go/blob/test-env-file/src/debugAdapter/goDebug.ts#L218). GoDebug is always loading everything `process.env`. Am I missing something?